JWBestPracticeApps/BasicPlayer/BasicPlayerViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ViewController.swift
+//  BasicPlayer
+//
+//  Created by David Perez on 19/11/20.
+//  Copyright © 2020 JWPlayer. All rights reserved.
+
+import UIKit
+import JWPlayerKit
+
+/**
+ Provides minimal setup for a JWPlayerViewController.
+ Create an instace of this player if you need a simple JWPlayerView.
+*/
+class BasicPlayerViewController: JWPlayerViewController {
+    private let videoUrlString = "https://playertest.longtailvideo.com/adaptive/oceans/oceans.m3u8"
+    private let posterUrlString = "https://d3el35u4qe4frz.cloudfront.net/bkaovAYt-480.jpg"
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Set the view's backgroundColor to .black for better contrast.
+        view.backgroundColor = .black
```

Comment:
Not relevant to this PR, but should we default the view's color to black?

Reply:
Ah good catch, I forgot about this. This basic BPA was made some time ago when the default idle background color was not black. This would be updated to be removed in the BasicBestPractice app ticket.